pkg/yurtctl/cmd/convert/convert.go

```diff
@@ -103,13 +106,16 @@ func NewConvertCmd() *cobra.Command {
 		Short: "Converts the kubernetes cluster to a yurt cluster",
 		Run: func(cmd *cobra.Command, _ []string) {
 			if err := co.Complete(cmd.Flags()); err != nil {
-				klog.Fatalf("fail to complete the convert option: %s", err)
+				klog.Errorf("fail to complete the convert option: %s", err)
```

Comment:
`klog.Fatalf` will print then calls os.Exit(255).
so why do we update this?

Reply:
After we upgraded `klog` to v2, `klog.Fatalf` will now print a verbose stack trace, which is not appropriate for a command-line tool, so I change `Fatalf` to `Errorf` and give it a non-zero exit code.

The exit code 1 here is referenced from codes from `kubeadm`:
https://github.com/kubernetes/kubernetes/blob/0153febd9f0098d4b8d0d484927710eaf899ef40/cmd/kubeadm/app/util/error.go#L31